docs/docs/cmd/tenant/report/report-o365activationsusercounts.md

```diff
@@ -0,0 +1,19 @@
+# tenant report office365activationsusercounts
```

Comment:
This file should be named `report-office365activationsusercounts.md`. With the current name it's not being loaded when requesting help for the command using the `-h` flag.